spec('failed', function() {
  throw new Error('test');
});
